deploy/charts/harvester/values.yaml

```diff
@@ -418,7 +420,7 @@ harvester-load-balancer:
   image:
     imagePullPolicy: Always
```

Comment:
We don't overwrite the imagePullPolicy in https://github.com/harvester/harvester-installer/blob/master/pkg/config/templates/rancherd-10-harvester.yaml#L119-L120
We need to set it to IfNotPresent

Reply:
resolved by https://github.com/harvester/harvester/compare/fc4876635550550c7119ad07b1cb8c71c876130a..5643f680beb90ae6a52bf4d9c536bd1f7880bc1e, good catch, thanks.